src/tools/genVersionHeader.pl

```diff
@@ -82,6 +88,7 @@
         $opt_V = $result;
         $vcs = 'Git';
     }
+    $cv = `git show -s --format=%ci HEAD`;
```

Comment:
For completeness, this prints something like `2020-05-05 16:59:56 -0500` circa Git 2.30.2.